imports/ui/case/case-details.jsx

```diff
@@ -123,6 +123,19 @@ class CaseDetails extends Component {
     </InfoItemContainer>
   )
 
+  renderPrioritySeverityLine = () => (
+    <InfoItemContainer>
+      <div className='flex'>
+        <div className='flex-grow'>
+          {infoItemMembers('Priority:', 'P3')}
+        </div>
+        <div className='flex-grow'>
+          {infoItemMembers('Severity:', 'Normal')}
+        </div>
+      </div>
+    </InfoItemContainer>
+  )
+
```

Comment:
@sophieyang0813 maybe I'm not reading your code correctly but it looks like you're 'forcing' the priority to be P3 and the severity to be 'Normal'
I don't understand why you'd want to do that...

The Priority and Severity should always be retrieved from the source of truth for the cases i.e the BZFE

Reply:
Thank you for pointing that out 😄😄 I didn't change that part after connecting default values and BZFE